apps/ledger-live-mobile/e2e/specs/speculos/addAccount.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { Application } from "../../page";
+
+let app: Application;
+
+const currencies = [
+  { currency: "bitcoin", nanoApp: "Bitcoin", tmsLink: "B2CQA-101" },
+  { currency: "ethereum", nanoApp: "Ethereum", tmsLink: "B2CQA-102" },
+];
+
+describe("Add accounts", () => {
+  beforeAll(async () => {
+    app = await Application.init("onboardingcompleted");
+    await app.portfolio.waitForPortfolioPageToLoad();
+  });
+
+  currencies.forEach(({ currency, nanoApp, tmsLink }) => {
+    let deviceName: string;
+
+    $TmsLink(tmsLink);
```

Comment:
seems to be specific to IntelliJ Idea, are we sure it works correctly with other IDE/Code editors ?

Reply:
Not sure but I am using VS Code and it is working for me. I referred the `jest-allure2-reporter` plugin documentation [here](https://wix-incubator.github.io/jest-allure2-reporter/docs/features/links/#tms-links)